NEXT_CHANGELOG.md

```diff
@@ -205,13 +207,14 @@ traffic_shaping:
       min_per_sec: 10
       ttl: 10s
       retry_percent: 0.2
+      retryable_mutations: false
```

Comment:
Suggest `retryable_mutations` -> `retry_mutations`.

Reply:
fixed in c47aafd